### PR TITLE
#122_Refine_release_candidate_2776_2776_2777_2762

### LINKED ISSUE #122: [3.35] [3.1.190618-rc1] Refine release candidate
Some smaller tasks

 - [x] [0.25] https://github.com/bitshares/bitshares-ui/issues/2780
 - [x] [0.5] https://github.com/bitshares/bitshares-ui/issues/2778
 - [x] [0.25] https://github.com/bitshares/bitshares-ui/issues/2777
 - [x] [0.1] https://github.com/bitshares/bitshares-ui/issues/2775
 - [x] [0.75] https://github.com/bitshares/bitshares-ui/issues/2776
 - [x] [1] https://github.com/bitshares/bitshares-ui/issues/2762

@Metizik @VerevkinAlexander this would be considered priority to get the release candidate done.

### PR DESCRIPTION
**Pull request by VerevkinAlexander**
<h2>General</h2>
Closes #122 

- Changed Input to Search input components
- Added sigma simbol instead of "cumulative"
- Added htlc header
- Increased "delete"  button tap area

<h2>General</h2>
Please make sure the following is done:

- Pull request is onto develop
- If you haven't already, have a look at
  - https://github.com/bitshares/bitshares-ui/blob/develop/CODE_OF_CONDUCT.md
  - https://github.com/bitshares/bitshares-ui/blob/develop/CONTRIBUTING.md

<h2>Code Preparation</h2>

_Please review all your changes one last time before committing_

- [ ] Check for unused code
- [ ] No unrelated changes are included
- [ ] None of the changed files are reformatting only
- [ ] Code is self explanatory or documented
- [ ] All written text is properly translated (english language)

<h2>Testing</h2>

_The branch has been tested on the following browsers (desktop and mobile view)_

- [ ] Chrome 
- [ ] Opera
- [ ] Firefox
- [ ] Safari

<h2>User interface changes</h2>

_Delete this section if there weren't any UI changes. Please make sure you tested your changes in all themes_

- [ ] Dark
- [ ] Light
- [ ] Midnight

_Please provide screenshots/licecap of your changes below_

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
I've adjsuted this PR and streamline the SearchInput. Please make it such that when user clicks on clear the text input is focused afterwards.

File: app/components/Explorer/Witnesses.jsx [hunk 1]
```diff
@@ -506,18 +506,12 @@ class Witnesses extends React.Component {
                                 placeholder={counterpart.translate(
                                     "explorer.witnesses.filter_by_name"
                                 )}
-                                value={this.state.filterWitness}
```
Comment (VerevkinAlexander) [inline app/components/Explorer/Witnesses.jsx]: @sschiessl-bcp Did you delete it accidentally?

**Comment by VerevkinAlexander** [comment]
I`ve finished plz review it

File: app/components/Explorer/Witnesses.jsx [hunk 2]
```diff
@@ -294,8 +294,8 @@ class WitnessList extends React.Component {
                         ? -1
                         : a.lastConfirmedBlock.timestamp <
                           b.lastConfirmedBlock.timestamp
-                            ? 1
-                            : 0;
+                        ? 1
```
Comment (sschiessl-bcp) [inline app/components/Explorer/Witnesses.jsx]: There is something in your prettier hook that creates those reformats, or maybe your IDE?

File: app/components/Explorer/Witnesses.jsx [hunk 3]
```diff
@@ -503,9 +503,7 @@ class Witnesses extends React.Component {
                             </div>
 
                             <SearchInput
-                                placeholder={counterpart.translate(
```
Comment (sschiessl-bcp) [inline app/components/Explorer/Witnesses.jsx]: filter_by_name had a different translation as compared to the default, fixed it

**Comment by sschiessl-bcp** [comment]
Fixed, accepted and forwarded

**Pull request closed without merging**